react-frontend/src/components/Resources/fordevelopers.js

```diff
@@ -4,7 +4,7 @@ import InfoCard from '../Home/infocard';
 
 function ForDevelopers() {
   return (
-    <div className>
```

Comment:
this error should never have gotten into develop.  It lights up the console with errors